api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -466,6 +468,25 @@ default void playSound(final @NotNull Sound sound) {
   default void playSound(final @NotNull Sound sound, final double x, final double y, final double z) {
   }
 
+  /**
+   * Plays a sound from an emitter, usually an entity.
+   *
+   * <p>
+   *   Sounds played using this method will follow the emitter unless the sound is a custom sound.
+   *   In this case the sound will be played at the location of the emitter and will not follow them.
+   * </p>
+   *
+   * <p><b>Note</b>: Due to <a href="https://bugs.mojang.com/browse/MC-138832">MC-138832</a>, the volume and pitch may be ignored when using this method.</p>
+   *
+   * @param sound a sound
+   * @param emitter an emitter
+   * @see Sound
+   * @see Sound.Emitter
```

Comment:
```suggestion
   * @see Sound.Emitter#self() to make a sound follow the receiver
```

(no need to reference the specific classes, they're already linked because of being method parameters)

Reply:
Yeah I only included this `@see` because every other method in `Audience` has one for method params or not. Also, this would make the text for the link into `to make a sound follow the receiver`, is this what you want? Happy to just add it into the body of the docs if not.

Reply:
yeah, that should change -- I've got the commit locally to remove some of the especially redundant `@see`s, and will push once this PR makes it in.

oops -- didn't realize that. Probably the best choice is `@see <code>Sound.Emitter.self()</code> to make a sound that follows the receiver`?